tests/test_api_test.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 by Christian <c.hagenest@pm.me>
+# Copyright (C) 2020 by Thomas Lindner <tom@dl6tom.de>
+#
+# SPDX-License-Identifier: 0BSD
+
+from fastapi import status
+
+
+def test_test_read_all_unauthorized(client, hood_id):
```

Comment:
Why the double "test_" in the beginning? o.0

Reply:
All test functions start with `test_`, so that pytest discovers them. In this case the API endpoint is named 'test' too, so the test function starts with `test_test_` for naming consistency.